packages/app-config/src/config/giving-universe/appConfig.development.ts

```diff
@@ -1,3 +1,51 @@
+import type { AppConfig, AuthTypes } from "@cfce/types"
+import appConfigBase from "../appConfigBase"
+
+const appConfig: AppConfig = {
+  ...appConfigBase,
+}
+
+// Override siteInfo
+appConfig.siteInfo = {
+  ...appConfig.siteInfo,
+  title: "Tests",
```

Comment:
Was Tests supposed to have its own app config, or use the giving-universe app config?

Reply:
Its own just in case we need to test stuff from different apps like give credits, etc that we may change at will without affecting giving universe config

Reply:
For some reason giving-universe config was not compiling and had to copy tests over universe to make it work